back/app/policies/initiative_policy.rb

```diff
@@ -38,6 +35,7 @@ def create?
   end
 
   def show?
+    return false if !can_moderate? && !owner? && InitiativeStatus::APPROVAL_CODES.include?(record.initiative_status.code)
     return true if active? && owner?
     return true if active? && can_moderate?
```

Comment:
This is getting hard to understand , imo.

I'll probably refactor to if/else statements, but for now just getting it working.